data/quarg/quarg missions.txt

```diff
@@ -325,3 +325,150 @@ mission "Quarg Pug Arfecta Warning"
 			`	As you make your way through the ring, you notice occasional glares from the Quarg onboard the ring.`
 				decline
+
+
+
+mission "Ask Quarg About Coalition Late"
+	invisible
+	landing
+	deadline 1
+	to offer
+		has "Ask Quarg About Coalition Early: offered"
+		or
+			has "joined the heliarchs"
+			has "joined the lunarium"
+	source
+		attributes "quarg"
+		attributes "station"
+	on offer
+		conversation
+				branch heliarch
+					has "joined the heliarchs"
+				branch lunarium
+					has "joined the lunarium"
+			label heliarch
+			`You've obtained the rank of Heliarch, the group who to this day remain hostile to the Quarg, but despite that you were allowed to land here on <origin>, whether because the Quarg simply don't know that you've joined the Heliarchs, or don't care enough to stop you from coming to their rings. Would you like to ask the Quarg for their side of the story, in light of what you've learned from the Heliarchs?`
+			choice
+				`	(Yes.)`
+					goto yes
+				`	(Not now.)`
+					defer
+				`	(No, I'm not interested.)`
+					decline
+			label lunarium
+			`You've joined the Lunarium in Coalition space, and even served as a messenger to bring the Quarg some information about the Lunarium itself, in the hopes that they might provide some help. Would you like to ask the Quarg for their side of the story, in light of what you've learned from the Lunarium and from your visit to the Quarg with Oobat?`
+			choice
+				`	(Yes.)`
+					goto yes
+				`	(Not now.)`
+					defer
+				`	(No, I'm not interested.)`
+					decline
+			label yes
+			`	You're never quite sure who to speak with when it comes to the Quarg, but you're instinctively drawn to one that stands out: you judge it to be an older member of their species, its thin scales covering its fingers, and the more bony parts of its head have turned to a bright, silvery shade. Also unlike most Quarg you've seen, it is wearing a pale, faded white robe, with patches of red-colored chains (that somehow perfectly transition into fabric again) becoming more and more prominent closer to the top, culminating in a firm, crimson hood, covering its head. Most surprising of all for a Quarg, however, is the large scar taking up much of the left side of its face, completely covering where its left eye should be.`
+			`	It is completely static, with a lost gaze focused on the ceiling here until you approach it. It doesn't squat down to meet your eye level, and says nothing, simply gazing at you with its one eye, colored a bright gold.`
+			choice
+				`	"Hello, I'm <first> <last>. What can you tell me about the Coalition?"`
+				`	"Hello. Is it true that the Heliarchs defeated you and stole your ringworlds?"`
+			`	It continues to stare at you for a while, without as much as blinking. It looks over to a bench in one of the quieter areas here, and gestures for you to follow. As you two sit down, it finally speaks up.`
```

Comment:
```suggestion
			`	It continues to stare at you for a while, without as much as blinking. It looks over to a bench in one of the quieter areas here, and gestures for you to follow. As you two sit down it finally speaks up.`
```
These three sentences all have the same structure, which looks off. The answer is, of course, commas.